CMakeLists.txt

```diff
@@ -583,6 +583,13 @@ if (TILEDB_TESTS)
   endif()
 
   add_subdirectory(test/regression)
+
+  # Add custom target 'check'
```

Comment:
Since you're touching this code, it's time to correct this comment, which is completely useless since all it does is to repeat what's apparent from the syntax. What would be useful is to list all kinds of tests that this target runs. For example, it builds all the dependencies of target `tests`. Does it actually run all of them now? That's what documentation is for.

Reply:
Yes it now runs all tests. I updated the comment.